localstack/services/events/event_bus.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+
+from localstack.aws.api.events import Arn, EventBusName, TagList
+from localstack.services.events.models_v2 import EventBus, RuleDict
+
+
+class EventBusService:
+    def __init__(
+        self,
+        name: EventBusName,
+        region: str,
+        account_id: str,
+        event_source_name: Optional[str] = None,
+        tags: Optional[TagList] = None,
+        policy: Optional[str] = None,
+        rules: Optional[RuleDict] = None,
+    ):
+        self.event_bus = EventBus(
+            name,
+            region,
+            account_id,
+            event_source_name,
+            tags,
+            policy,
+            rules,
+        )
+
+    @property
+    def arn(self):
+        return self.event_bus.arn
+
+
+EventBusServiceDict = dict[Arn, EventBusService]
```

Comment:
What value does this class add? Is this a preparation for some future part of the provider, or just to match the `RuleService`?

Reply:
Yes, this is mostly preparation for the future implementation dealing with persistence. My goal is to add an additional abstraction on top of the AccountRegionBundle, that combines state that is persistent and logic into an object, so that the provider only sees this object.

Reply:
The question was more related to this specific class, since it does not feature any additional logic other than holding the event bus state object. As discussed in slack, it is part a remnant of early implementation, and will be used when more logic regarding event busses arrives. Fine with me to keep it for now.